.NET/Microsoft.Recognizers.Text.DateTime.Tests/Spanish/TestDateParser.cs

```diff
@@ -10,6 +10,13 @@ public class TestDateParser
         readonly IDateTimeParser parser;
         readonly BaseDateExtractor extractor;
 
+        [ClassCleanup]
+        public static void ClassCleanup()
+        {
+            TestWriter.Close("Spa", typeof(BaseDateParser));
+        }
+
```

Comment:
Nit

Reply:
Moved to string constants (e.g.: TestCulture.English to represent "English")